### PR TITLE
Update pl-pl.json

### PR DESCRIPTION
**Pull request by andrewczm**
Some minor corrections in the Polish translation

### CODE REVIEW

File: src/lang/pl-pl.json [hunk 1]
```diff
@@ -367,10 +367,10 @@
 	"remove": "Usuń",
 	"quicktools:command-palette": "Paleta poleceń",
 	"default file encoding": "Domyślne kodowanie plików",
-	"remove entry": "Czy na pewno chcesz usunąć '{nazwa}' z zapisanych ścieżek? Należy pamiętać, że usunięcie go nie spowoduje usunięcia samej ścieżki.",
-	"delete entry": "Potwierdź usunięcie: '{nazwa}'. Tej akcji nie można cofnąć. Kontynuować?",
+	"remove entry": "Czy na pewno chcesz usunąć '{name}' z zapisanych ścieżek? Należy pamiętać, że usunięcie go nie spowoduje usunięcia samej ścieżki.",
+	"delete entry": "Potwierdź usunięcie: '{name}'. Tej akcji nie można cofnąć. Kontynuować?",
 	"change encoding": "Czy ponownie otworzyć '{file}' z kodowaniem '{encoding}'? Ta czynność spowoduje utratę wszelkich niezapisanych zmian dokonanych w pliku. Czy chcesz kontynuować ponowne otwieranie?",
-	"reopen file": "Czy na pewno chcesz ponownie otworzyć '{plik}'? Wszelkie niezapisane zmiany zostaną utracone.",
+	"reopen file": "Czy na pewno chcesz ponownie otworzyć '{file}'? Wszelkie niezapisane zmiany zostaną utracone.",
```
Comment (bajrangCoder) [inline src/lang/pl-pl.json:373]: Why using english in translation

File: src/lang/pl-pl.json [hunk 2]
```diff
@@ -367,10 +367,10 @@
 	"remove": "Usuń",
 	"quicktools:command-palette": "Paleta poleceń",
 	"default file encoding": "Domyślne kodowanie plików",
-	"remove entry": "Czy na pewno chcesz usunąć '{nazwa}' z zapisanych ścieżek? Należy pamiętać, że usunięcie go nie spowoduje usunięcia samej ścieżki.",
-	"delete entry": "Potwierdź usunięcie: '{nazwa}'. Tej akcji nie można cofnąć. Kontynuować?",
+	"remove entry": "Czy na pewno chcesz usunąć '{name}' z zapisanych ścieżek? Należy pamiętać, że usunięcie go nie spowoduje usunięcia samej ścieżki.",
+	"delete entry": "Potwierdź usunięcie: '{name}'. Tej akcji nie można cofnąć. Kontynuować?",
```
Comment (bajrangCoder) [inline src/lang/pl-pl.json:371]: Why english words?

**Review by bajrangCoder — CHANGES_REQUESTED** [review]

File: src/lang/pl-pl.json (hunk 2 above)
Comment (andrewczm) [reply src/lang/pl-pl.json:371]: I thought it was a variable that could not be translated. If it can be translated then I will correct it soon.

Comment (andrewczm) [reply src/lang/pl-pl.json:371]: corrected

Comment (bajrangCoder) [reply src/lang/pl-pl.json:371]: Yeah , sorry those are variables (I was confused)

Comment (bajrangCoder) [reply src/lang/pl-pl.json:371]: No need to correct it

Comment (andrewczm) [reply src/lang/pl-pl.json:371]: ok, I restored the original phrases for the variables, they are in English now. You can merge it.

**Review by bajrangCoder — APPROVED** [review]

**Pull request merged**